README.md

```diff
@@ -1,5 +1,5 @@
 # DOS Wasm X
-This is a browser based DOS emulator designed around ease of use and stability. It is based on the newer DosBox-X codebase and thus supports both Windows 95 and Windows 98 installations. However if you just want to use DOS applications and games you can stay in DOS mode. To begin using it simply drag and drop any application or game files onto the emulator. You can then save your hard disk with the click of a button or just exit if you want to discard your changes. I went with a simple and clean interface to try and make it approachable and non-intimidating. The hard disk saves directly in your browser so you can come back later and continue where you left off. It's like your own personal virtual machine on the web!
+This is a browser based DOS emulator designed around ease of use and stability. It is based on the newer DosBox-X codebase and thus supports Windows 3.x, Windows 9x, and Windows NT installations. However if you just want to use DOS applications and games you can stay in DOS mode. To begin using it simply drag and drop any application or game files onto the emulator. You can then save your hard disk with the click of a button or just exit if you want to discard your changes. I went with a simple and clean interface to try and make it approachable and non-intimidating. The hard disk saves directly in your browser so you can come back later and continue where you left off. It's like your own personal virtual machine on the web!

```

Comment:
I would leave this part the way it is currently as this is just an intro. With the limitations outlined I don't know if we should highlight it as "supported" but the detailed docs can expand upon it per my other comment